marklogic-data-hub-central/ui/src/components/queries/managing/manage-query.tsx

```diff
@@ -0,0 +1,224 @@
+import React, { useState, useEffect, useContext } from 'react';
+import { Modal, Button, Table } from 'antd';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPencilAlt, faFileExport, faLink, faTrashAlt, faListOl } from "@fortawesome/free-solid-svg-icons";
+import { UserContext } from '../../../util/user-context';
+import { queryDateConverter } from '../../../util/date-conversion';
+import EditQueryDialog from './edit-query-dialog/edit-query-dialog'
+import { SearchContext } from '../../../util/search-context';
+import styles from './manage-query.module.scss';
+import { fetchQueries, updateQuery, removeQuery } from '../../../api/queries'
+
+
+const QueryModal: React.FC = () => {
+
+    const [queries, setQueries] = useState([]);
+    const { handleError, resetSessionTime } = useContext(UserContext);
+    const { applyQuery } = useContext(SearchContext);
+    const [mainModalVisibility, setMainModalVisibility] = useState(false);
+    const [editModalVisibility, setEditModalVisibility] = useState(false);
+    const [deleteModalVisibility, setDeleteModalVisibility] = useState(false);
+    const [selectedQuery, setSelectedQuery] = useState({});
+    const [queryName, setQueryName] = useState('');
+    const data = new Array();
+
+    useEffect(() => {
+        getQueries();
+    }, []);
+
```

Comment:
@timur-isangulov, Any reason for using two useEffects calling the same function ?

Reply:
Not needed, removed.